### PR TITLE
Second review round, i.e. with OSF models etc.

### PR DESCRIPTION
**Pull request by chrzyki**

### CODE REVIEW

**Comment by chrzyki** [comment]
As usual, feel free to change/adapt as needed. No need to reply to the things that were fixed in recent commits!

**Comment by chrzyki** [comment]
See https://github.com/dlce-eva/papers/pull/2.

File: scripts/00_setup.R [hunk 1]
```diff
@@ -19,4 +19,6 @@ source("03_PriorModel.R")
 
 source("05_ModelConvergence.R")
 source("06_BayesViz.R")
+
+# TODO: This name has been changed 07_utils.R
```
Comment (FredericBlum) [inline scripts/00_setup.R:23]: done

File: scripts/03_PriorModel.R [hunk 2]
```diff
@@ -16,7 +16,9 @@ data <- read_tsv('data.tsv') %>%
          utt_initial = as.factor(utt_initial))
 
 # If necessary, set path to specific cmdstan installation
-set_cmdstan_path(path = "/data/tools/stan/cmdstan-2.32.2/")
+# TODO: I think a default isn't necessary here, just the comment above and
```
Comment (FredericBlum) [inline scripts/03_PriorModel.R:19]: done

File: scripts/03_PriorModel.R [hunk 3]
```diff
@@ -78,6 +80,8 @@ ggsave("images/prior_langAll.png", prior_lang_all, scale=1,
 
 raw_durations <- data %>% .$Duration
 
+# TODO: Do filenames differ from the ones stored on OSF?
+# I.e. OSF: pred_prior.rds, here: prior_pred.rds.
```
Comment (FredericBlum) [inline scripts/03_PriorModel.R:84]: changed for coherency

File: scripts/04_FinalModel.R [hunk 4]
```diff
@@ -8,6 +8,7 @@ data <- read_tsv('data.tsv') %>%
          word_initial = as.factor(word_initial),
          utt_initial = as.factor(utt_initial))
 
+# TODO: See other set_cmdstan_path comment.
```
Comment (FredericBlum) [inline scripts/04_FinalModel.R:11]: done

File: scripts/05_ModelConvergence.R [hunk 5]
```diff
@@ -102,6 +102,8 @@ if (file.exists("models/post_pred.rds")) {
 duration_vals <- data %>% pull(Duration)
 group_init <- data %>% pull(initial)
 
+# TODO: Can't reproduce that with the latest post_pred.rds from OSF:
+# Error in sim_data[1:5, ] : subscript out of bounds
```
Comment (FredericBlum) [inline scripts/05_ModelConvergence.R:106]: fixed

File: scripts/06_BayesViz.R [hunk 6]
```diff
@@ -14,7 +14,9 @@ library(xtable)
 ###################################################################
 model <- readRDS(file="models/cl_final.rds")
 
-languages <- read_csv('../doreco/cldf/languages.csv') %>% 
+# TODO: Another hint might be useful in case users aren't following the exact
```
Comment (FredericBlum) [inline scripts/06_BayesViz.R:17]: added

File: scripts/06_BayesViz.R [hunk 7]
```diff
@@ -297,6 +301,7 @@ speaker_plot_word <- speaker_variation %>%
   scale_y_continuous(limits=c(-0.3, 0.32), name="Estimated value") +
   theme(legend.position='none')
 
+# TODO: For a number of tiles, the language name is too long to fit into the title bar.
```
Comment (FredericBlum) [inline scripts/06_BayesViz.R:304]: Fixed

File: scripts/06_BayesViz.R [hunk 8]
```diff
@@ -315,5 +320,6 @@ speaker_plot_utt <- speaker_variation %>%
   scale_y_continuous(limits=c(-0.32, 0.32), name="Estimated value") +
   theme(legend.position='none')
 
+# TODO: Likewise, language names too long.
```
Comment (FredericBlum) [inline scripts/06_BayesViz.R:323]: Fixed

File: scripts/03_PriorModel.R (hunk 3 above)
Comment (FredericBlum) [reply scripts/03_PriorModel.R:84]: in a previous commit

File: scripts/03_PriorModel.R [hunk 9]
```diff
@@ -101,6 +105,8 @@ prior_overlay <- ppc_dens_overlay(raw_durations, priorsim_durations[1:8, ],
 
 prior_overlay$scales$scales[[1]]$labels <- c("data", "prior")
 
+# TODO: Not sure I like the way the two plots are stacked with their axis labels.
+# Include density axis on at least y for the bottom plot to make this clearer?
```
Comment (FredericBlum) [inline scripts/03_PriorModel.R:109]: Supplementary plot that does not need to be perfect

**Review by FredericBlum — COMMENTED** [review]
The most recent release candidate addresses all of the mentioned points.

**Comment by chrzyki** [comment]
Looks good to me, thanks for taking care of my comments!

**Pull request closed without merging**